### DISCUSSION TITLE
リリースノートページで、ダークモードのテーマが適用されない

### DISCUSSION THREAD

**Discussion by fussy113** [increments/qiita-discussions#611] [要望・アイデア・不具合]
システムテーマをダークにして、リリースノートページを表示した時、ダークモードの表示が適用されていないようでした。
2段階認証のバックアップコードについての通知のラベルに対してはダークモードが適用され、見づらくなってしまっていました。
(最新版のGoogle Chromeにて、確認しています)

https://qiita.com/release-notes

<img width="1200" alt="リリースノートページの表示" src="https://github.com/increments/qiita-discussions/assets/15154105/ec7bf7ed-f06b-482a-8934-3e1afc45d052">

**Comment by tanaka-asami415** [comment]
@fussy113 
ご報告ありがとうございます。
リリースノートページはまだダークモードに対応できていないのですが、将来的に対応する予定です。
通知のラベルが見にくくなってしまっているのは、別途対応いたします🙇

**Reply by tanaka-asami415** [reply]
@fussy113 
リリースノートのダークテーマ対応はまだ先になりそうなため、一旦通知のラベルを修正いたしました！

**Reply by tanaka-asami415** [reply]
@fussy113 
お待たせしました！
リリースノートページのダークテーマも対応いたしました🚀
https://qiita.com/release-notes/2024-03-29

この度はご意見をいただきありがとうございました。
